assets/js/script.js

```diff
@@ -168,22 +169,21 @@ function render() {
   $('.js-tab').html('')
   $('.js-preview').html('')
   $('.js-tab').removeClass('active')
-  $('.js-tab').parent().removeClass('active')
+  $('.js-tab-back').removeClass('active')
   $('.js-swatch').removeClass('active')
   $('.js-loading').hide()
 
   // render bag color
+  $('.js-bag').css({ backgroundImage: "url(" + (window.baseUrl || '') + "assets/img/bags/" + state.bag.style + "-" + state.bag.color + ".png)" })
 
-  $('.js-bag').css({ backgroundImage: "url(" + (window.baseUrl || '') + "assets/img/bags/danny-" + state.bag + ".png)" })
-
+  // render tabs
   $('.js-tab').each(function() {
-      if ($(this).data('index') != -1) {
+      if ($(this).data('index') == -1) {
+        $(this).html('All')
```

Comment:
I was saying you can `return $(this).html('All')` here, then you don't need the `else` on the next line, and you save some indentation for readability's sake. 

again, totally not a big deal, you don't have to change it.